backend/src/main/resources/db/migration/V1__create_tables.sql

```diff
@@ -0,0 +1,146 @@
+                                            id bigint PRIMARY KEY,
+                                            value text NOT NULL
+);
+
+CREATE TABLE healthcare_provider (
+                                            institution_id bigint NOT NULL,
+                                            location_id bigint NOT NULL,
+                                            administrative_district text,
+                                            care_form text,
+                                            care_type text,
+                                            city text NOT NULL,
+                                            code text NOT NULL,
+                                            district text NOT NULL,
+                                            district_code text NOT NULL,
+                                            email text,
+                                            fax text,
+                                            house_number text NOT NULL,
+                                            hq_city text,
+                                            hq_district text,
+                                            hq_district_code text,
+                                            hq_house_number text,
+                                            hq_postal_code text,
+                                            hq_region text,
+                                            hq_region_code text,
+                                            hq_street text,
+                                            ico text NOT NULL,
+                                            institution_type text NOT NULL,
+                                            lat double precision,
+                                            lawyer_form_code text,
+                                            layer_form text,
+                                            lng double precision,
+                                            person_type text NOT NULL,
+                                            person_type_code text NOT NULL,
+                                            phone_number text,
+                                            postal_code text NOT NULL,
+                                            region text NOT NULL,
+                                            region_code text NOT NULL,
+                                            specialization text NOT NULL,
+                                            street text,
+                                            substitute text,
+                                            title text NOT NULL,
+                                            website text
+);
+
+CREATE TABLE healthcare_provider_category (
+                                                     institution_id bigint NOT NULL,
+                                                     location_id bigint NOT NULL,
+                                                     id bigint NOT NULL
+);
+
+CREATE SEQUENCE hibernate_sequence
```

Comment:
Where do we use this sequence? I didn't find it in the code.

Reply:
We don't use it explicitly, but hibernate does, it uses this sequence to get a new ID value when new record is inserted, this configuration tells hibernate to do so:
```
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    val id: Long = 0,
```

Reply:
I'm just thinking, shall we rewrite the strategy to sequence, so we'd have it more under our control? We can do it in a separate task

Reply:
Ok. I am used to the named sequence in an annotation. One for all tables is a little bit crazy for me :).
On the other hand, DB such as MySQL has an auto-increment column. I know, this is Postgres ... :).